substrate/frame/conviction-voting/src/lib.rs

```diff
@@ -170,6 +170,10 @@ pub mod pallet {
 		Delegated(T::AccountId, T::AccountId),
 		/// An \[account\] has cancelled a previous delegation operation.
 		Undelegated(T::AccountId),
+		/// An account that has voted
+		VoteEnacted { who: T::AccountId, vote: AccountVote<BalanceOf<T, I>> },
```

Comment:
```suggestion
		Voted { who: T::AccountId, vote: AccountVote<BalanceOf<T, I>> },
```

Enacted sounds to me more final.